eventbridge-schedule-to-cloudwatch-alarm-terraform/README.md

```diff
@@ -0,0 +1,56 @@
+# AWS Service 1 to AWS Service 2
```

Comment:
Please be more descriptive

Reply:
Sorry, I forgot to update this line. Should I use "EventBridge Scheduler for CloudWatch Alarm" for this?